src/ast.rs

```diff
@@ -1152,18 +1214,12 @@ mod tests {
                     "$schema",
                     "64bit",
                 ]}}});
+
         let mut tag: Tag = serde_json::from_value(data).unwrap();
-        tag.fix_properties();
-        if let Type::Object(object) = &tag.data_type {
-            let expected: HashSet<String> = ["valid_name", "renamed_value_0", "_64bit"]
-                .iter()
-                .map(|x| x.to_string())
-                .collect();
-            let actual: HashSet<String> = object.fields.keys().cloned().collect();
-            assert_eq!(expected, actual);
-            assert_eq!(expected, object.required.clone().unwrap());
-        } else {
-            panic!()
-        }
+        tag.normalize_properties(false);
+        assert_normalize(&tag, vec!["valid_name", "renamed_value_0", "_64bit"]);
+
+        tag.normalize_properties(true);
+        assert_normalize(&tag, vec!["valid_name", "renamed_value_0", "_64bit"]);
```

Comment:
This case covers the behavior of the `heck` library, which will ignore punctuation outside of word boundaries e.g. `_64bit` is treated as `64bit`. This double checks that the casing library doesn't change this particular behavior.

Reply:
Maybe add that as a code comment? A PR comment and this knowledge will get lost